django_mesh/tests/test_models.py

```diff
@@ -16,6 +16,19 @@
 
 # Test imports
 from .util import BaseTestCase
+import markdown, textile
+import requests
+from pyembed.markdown import PyEmbedMarkdown
+from pyembed.core import PyEmbed
+from django_mesh.models import _Abstract
+from .youtube_data import get_mock
+from django.core.urlresolvers import reverse
+
+try:
+    from mock import patch, Mock
```

Comment:
Since this is added as a test dependency, you can take it out of a try/except block.